packages/tree-widget/README.md

```diff
@@ -2,181 +2,28 @@
 
 Copyright © Bentley Systems, Incorporated. All rights reserved.
 
-The tree-widget-react package provides a controller class - `TreeWidgetControl` - which can be plugged into a 9-zone widget to
-provide access to the following features:
+The tree-widget-react package provides a UiProvider class - `TreeWidgetUiItemsProvider` - which can be passed into the `uiProviders` prop of the iTwin Viewer's `<Viewer />` component to provide access to the following features:
 
 - Spatial Containment Tree
 
 - Models Tree
 
 - Categories Tree
 
-The package also provides the component which underlies `TreeWidgetControl` - `TreeWidgetComponent` - which you can wrap within your own custom widget controller and pass in your own custom trees to display.
+The package also provides the widget controller class - `TreeWidgetControl` - as well as the underlying component - `TreeWidgetComponent` - which you can wrap within your own custom UiProvider and pass in your own custom trees to display.
```

Comment:
tehcnically can't use `TreeWidgetControl` in a UiProvider(ui-2.0), that's more for adding directly into a FrontstageDef (ui-1.0)

I would probably just drop the mention of the Control class.
```suggestion
The package also provides the underlying component - `TreeWidgetComponent` - which you can wrap within your own custom UiProvider and pass in your own custom trees to display.
```